src/random_lcg.F90

```diff
@@ -53,6 +54,21 @@ function prn() result(pseudo_rn)
   end function prn
 
 !===============================================================================
+! GET_PRN_AHEAD generates a pseudo-random number which is 'n' times ahead from
+! current seed.
+!===============================================================================
+
+  function get_prn_ahead(n) result(pseudo_rn)
```

Comment:
I'd prefer this be called simply `prn_ahead`. It's clear from the fact that it's a function that you're going to 'get' something.